ceci/config.py

```diff
@@ -111,6 +122,11 @@ def set_to_default(self):
         self._value = cast_value(self._dtype, self._default)
         return self._value
 
+    def set_default(self, default):
+        """Set the default value"""
+        self._default = default
+        self.set_to_default()
```

Comment:
Shoud changing the default value of a parameter also set that parameter to the default? That doesn't seem obvioue?

Reply:
okay, that's reasonable!

Reply:
Maybe set_default(self, default, preserve_value=False) would be better, as it defaults to change the value, but let's you keep it if you want.   
Totally up to you what you think is best.